src/Symfony/Component/Messenger/Transport/AmqpExt/AmqpTransportFactory.php

```diff
@@ -20,6 +20,8 @@
 /**
  * @author Samuel Roze <samuel.roze@gmail.com>
  */
+// by design this class encourages creating a separate connection per DSN
+// can't we do without, ie drop it and rely on the DI extension in fwb to do the work instead? See comment there.
```

Comment:
Likely should be part of the class docblock?

Reply:
I think this is some question to either accept or deny like an RFC, then modify the class if we agree ;).